src/Exception/NoContentResponseException.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Exception to send an empty response and exit the execution in the Contao workflow.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class NoContentResponseException extends AbstractResponseException
```

Comment:
I think that should be `EmptyResponseException` (even your comment says "send an empty response").